pkgs/development/python-modules/homf/default.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  # pytestCheckHook,
+  pythonOlder,
+
+  hatchling,
+  packaging,
+}:
+
+buildPythonPackage rec {
+  pname = "homf";
+  version = "1.0.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "duckinator";
+    repo = pname;
```

```suggestion
    repo = "homf";
```
This is a misuse of `pname`.